setup.py

```diff
@@ -41,6 +41,9 @@ def read(*parts):
 cmdclass = {'test': PyTest}
 cmdclass.update(versioneer.get_cmdclass())
 
+with open('requirements.txt') as f:
+    require = f.readlines()
+install_requires = [r.strip() for r in require]
```

Comment:
Happily, setuptools already ignores entries with a leading `#` so there's no need to filter them out here.